lib/community/digest2/personalizeData.js

```diff
@@ -82,8 +82,9 @@ const personalizeData = (user, data, opts = {}) => {
     },
     email_settings_url: Frontend.Route.userSettings() + clickthroughParams + '&expand=account',
     tracking_pixel_url: Analytics.pixelUrl('Digest', {userId: user.id, community: data.community_name}),
-    form_action_url: Frontend.Route.emailPostForm(),
```

Comment:
you can't rename this; it's still used by the other template version. well, you can, but you'll have to edit the other template version once this is deployed. if you want to do that, i would suggest creating a subtask in Asana for it.